orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -88,7 +89,8 @@ class SqlExecutionRepository(
   private val batchReadSize: Int = 10,
   private val stageReadSize: Int = 200,
   private val poolName: String = "default",
-  private val interlink: Interlink? = null
+  private val interlink: Interlink? = null,
+  private val executionRepositoryListener: ExecutionRepositoryListener? = null
```

Comment:
Does it make sense to make this a collection? That way you can invoke it in a `forEach` without needing a null check.

Reply:
Sure, that makes sense. I wasn't anticipating having multiples of these, but probably wouldn't hurt.